### PR TITLE
Feature/submenu icon

### PR DESCRIPTION
**Pull request by KVGrygoriev**
Fixes #[1747](https://github.com/smartdevicelink/sdl_core/issues/1747)

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Testing Plan
Test scripts PR : https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1898

### Summary
This PR contains the implementation of proposal https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0085-submenu-icon.md

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: app/controller/sdl/RPCController.js [hunk 1]
```diff
@@ -1229,6 +1262,12 @@ SDL.RPCController = Em.Object.create(
             return this.resultStruct;
           }
         },
+        SendHapticData: function(params) {
```
Comment (AKalinich-Luxoft) [inline app/controller/sdl/RPCController.js]: @KVGrygoriev please provide description for that

File: ffw/BasicCommunicationRPC.js [hunk 2]
```diff
@@ -243,7 +243,10 @@ FFW.BasicCommunication = FFW.RPCObserver
           } else {
             this.OnSystemRequest('PROPRIETARY');
           }
-          SDL.SettingsController.policyUpdateRetry();
+
+          if (FLAGS.ExternalPolicies === true) {
+            SDL.SettingsController.policyUpdateRetry();
```
Comment (AKalinich-Luxoft) [inline ffw/BasicCommunicationRPC.js]: @KVGrygoriev is these changes related to this feature? I suggest you to create a separate pull request with fix for policy flow and keep here only changes relevant for this feature

**Comment by AKalinich-Luxoft** [comment]
@KVGrygoriev please change base of this pull request to `develop`

File: ffw/BasicCommunicationRPC.js (hunk 2 above)
Comment (KVGrygoriev) [reply ffw/BasicCommunicationRPC.js]: @AKalinich-Luxoft  I didn't this changes. I think this was because of I was choose master branch for merge...now it changed to develop.
Check please again.
